src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
-  margin: 0;
+  margin: auto 0;
+}
+
+/* Linha em branco antes da regra */
+.html {
```

Comment:
You are trying to stylize a class that doesn't exists. Maybe is just `html` instead of `.html`?